src/components/CalendarPicker/calendarPickerPropTypes.js

```diff
@@ -1,41 +1,38 @@
 import PropTypes from 'prop-types';
+import moment from 'moment';
+import CONST from '../../CONST';
 
-const propTypes = {
-    /** A function that is called when the date changed inside the calendar component */
-    onChanged: PropTypes.func,
-
-    /** A function called when the date is selected */
-    onSelected: PropTypes.func,
-
-    /** A value initial of date */
+const calendarPickerPropType = {
+    /** An initial value of date */
     value: PropTypes.objectOf(Date),
 
-    /** A minimum date of calendar to select */
+    /** A minimum date (oldest) allowed to select */
     minDate: PropTypes.objectOf(Date),
 
-    /** A maximum date of calendar to select */
+    /** A maximum date (earliest) allowed to select */
     maxDate: PropTypes.objectOf(Date),
 
-    /** Callback function to call when year is pressed */
-    onYearPressed: PropTypes.func,
-
-    /** Default month to be set in the calendar picker */
-    defaultMonth: PropTypes.string,
-
     /** Default year to be set in the calendar picker. Used with navigation to set the correct year after going back to the view with calendar */
     defaultYear: PropTypes.string,
 
-    /** Callback function to run when pressing close button. Used with navigation to set the correct year after going back to the view with calendar */
-    onClosePressed: PropTypes.func,
+    /** A function that is called when the date changed inside the calendar component */
+    onChanged: PropTypes.func,
+
+    /** A function called when the date is selected */
+    onSelected: PropTypes.func,
+
+    /** Callback function to call when year is pressed */
+    onYearPressed: PropTypes.func,
 };
 
-const defaultProps = {
+const defaultCalendarPickerPropType = {
     value: new Date(),
-    minDate: null,
-    maxDate: null,
-    onChanged: null,
-    onSelected: null,
-    onYearPressed: null,
+    minDate: moment().year(CONST.CALENDAR_PICKER.MIN_YEAR).toDate(),
+    maxDate: moment().year(CONST.CALENDAR_PICKER.MAX_YEAR).toDate(),
+    defaultYear: null,
+    onChanged: () => {},
+    onSelected: () => {},
+    onYearPressed: () => {},
 };
 
-export {propTypes, defaultProps};
+export {calendarPickerPropType, defaultCalendarPickerPropType};
```

Comment:
NAB: We usually export those as `propTypes, defaultProps` and import them as `calendarPickerPropTypes, calendarPickerDefaultProps`.

Reply:
+1